src/taipy/rest/api/middlewares/_taipy_middleware.py

```diff
@@ -0,0 +1,38 @@
+# Copyright 2022 Avaiga Private Limited
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+# the License. You may obtain a copy of the License at
+#
+#        http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+# an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+# specific language governing permissions and limitations under the License.
+
+from functools import wraps
+from importlib import util
+
+from flask import request
+
+from taipy.core.common._utils import _load_fct
+
+
+def _taipy_middleware(f):
```

Comment:
I am not big fan of the name, but I don't have better name to propose at this point.